rules/snv.smk

```diff
@@ -95,6 +95,8 @@ rule snv:
         WORK_DIR = "{dataset}/variants/SNVs",
         LOCALSCRATCH = config.snv['localscratch'],
         SHORAH = config.applications['shorah'],
+        THREADS = config.snv['threads'],
```

Comment:
Why is the `params.THREADS` needed? you can simply use the directive `threads`

Reply:
Why: Because I am a complete beginner in python and snakemake? :sweat_smile: